scrapers/PornDudeCasting.yml

```
@@ -0,0 +1,82 @@
+name: PornDudeCasting
+performerByURL:
+  - action: scrapeXPath
+    url:
+      - porndudecasting.com/casting/
+    scraper: performerScraper
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - porndudecasting.com/casting/
+    scraper: sceneScraper
+xPathScrapers:
+  performerScraper:
+    performer:
+      Name: //h5[@class="model__head-name"]/text()
+      Gender:
+        fixed: "Female"
+      URL: //head/meta[@property="og:url"]/@content
+      Twitter:
+        selector: //div[@class='model__head desctop']//a[@class='card__user-social-link twitter']/@href
+        postProcess:
+          - replace:
+            - regex: ([\s\S]+)twitter.com/
+              with: ""
+      Instagram:
+        selector: //div[@class='model__head desctop']//a[@class='card__user-social-link instagram']/@href
+        postProcess:
+          - replace:
+            - regex: ([\s\S]+)instagram.com/
+              with: ""
+      Country: //ul[@class='model__info-list desctop']/li/span[text()='From:']/../span[2]/text()
+      Height:
+        selector: //ul[@class='model__info-list desctop']/li/span[text()='Height:']/../span[2]/text()
+        postProcess:
+          - replace:
+            - regex: \d'\D\d+\D+
+              with: ""
+            - regex: \D
+              with: ""
+      # To get Performerimages Uncomment below and enable CDP at the end of the File
+      #Image:
+        #selector: //div[@class="model__img-box small"]/img/@src
```

Seems like you can get this without CDP:
```suggestion
        #selector: //div[@class="model__img-box small"]/img/@data-src
```
And when developing scrapers it's usually a good idea to disable JavaScript on the page, you need the raw HTML, since Stash can not execute the JavaScript code like a browser or CDP can.

---

that works dont know how i did not see that